src/cmd-buildextend-gcp

```diff
@@ -111,7 +111,14 @@ def run_ore():
     checksum = sha256sum_file(build_tarball)
     size = os.path.getsize(build_tarball)
 
-    buildmeta['gcp'] = {'image': f"{base_name}-{args.build.replace('.', '-')}"}
+    gcp_name = f"{base_name}-{args.build.replace('.', '-')}"
+    gcp_path = f"{args.bucket}/{base_name}/{gcp_tarball_name}"
+    url = f"https://console.cloud.google.com/storage/browser/{gcp_path}"
```

Comment:
https://cloud.google.com/storage/docs/request-endpoints#access_to_public_objects

Reply:
the api reference is here https://cloud.google.com/compute/docs/reference/rest/v1/images/insert
`rawDisk.source`

Reply:
Fixed